include/sqlpp11/data_types/text/like.h

```diff
@@ -45,7 +45,7 @@ namespace sqlpp
 
     struct _alias_t
     {
-      static constexpr const char _literal[] = "like_";
+      static constexpr const char _literal[6] = "like_";
```

Comment:
It's necessarily because of half solved bug https://connect.microsoft.com/VisualStudio/feedback/details/2100550/-static-constexpr-const-char-literal-delta-not-working-inside-templated-struct